code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -82,6 +82,14 @@ RESTRICT_TYPE(/datum/antagonist/traitor)
 
 	return ..()
 
+/datum/antagonist/traitor/select_organisation()
+	var/chaos = pick(prob(ORG_PROB_HUNTER);ORG_CHAOS_HUNTER, prob(ORG_PROB_MILD);ORG_CHAOS_MILD, prob(ORG_PROB_AVERAGE);ORG_CHAOS_AVERAGE, prob(ORG_PROB_HIJACK);ORG_CHAOS_HIJACK)
```

Comment:
We are trying to move away from this `pick(2;"thing")` dynamic. Consider using something like `pickweight` instead.